apps/kg-explorer/src/app/pages/main-page/main-page.spec.ts

```diff
@@ -0,0 +1,758 @@
+        },
+      ],
+      distributions: [
+        {
+          id: 'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#2d-ftu-kidney-ascending-thin-loop-of-henle.svg',
+          label: "Raw data distribution '2d-ftu-kidney-ascending-thin-loop-of-henle.svg' file",
+          title: "Raw data distribution '2d-ftu-kidney-ascending-thin-loop-of-henle.svg' file",
+          downloadUrl:
+            'https://cdn.humanatlas.io/digital-objects/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2/assets/2d-ftu-kidney-ascending-thin-loop-of-henle.svg',
+          accessUrl:
+            'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#2d-ftu-kidney-ascending-thin-loop-of-henle.svg',
+          mediaType: 'image/svg+xml',
+        },
+        {
+          id: 'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#2d-ftu-kidney-ascending-thin-loop-of-henle.png',
+          label: "Raw data distribution '2d-ftu-kidney-ascending-thin-loop-of-henle.png' file",
+          title: "Raw data distribution '2d-ftu-kidney-ascending-thin-loop-of-henle.png' file",
+          downloadUrl:
+            'https://cdn.humanatlas.io/digital-objects/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2/assets/2d-ftu-kidney-ascending-thin-loop-of-henle.png',
+          accessUrl:
+            'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#2d-ftu-kidney-ascending-thin-loop-of-henle.png',
+          mediaType: 'image/png',
+        },
+        {
+          id: 'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#2d-ftu-kidney-ascending-thin-loop-of-henle.ai',
+          label: "Raw data distribution '2d-ftu-kidney-ascending-thin-loop-of-henle.ai' file",
+          title: "Raw data distribution '2d-ftu-kidney-ascending-thin-loop-of-henle.ai' file",
+          downloadUrl:
+            'https://cdn.humanatlas.io/digital-objects/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2/assets/2d-ftu-kidney-ascending-thin-loop-of-henle.ai',
+          accessUrl:
+            'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#2d-ftu-kidney-ascending-thin-loop-of-henle.ai',
+          mediaType: 'application/postscript',
+        },
+        {
+          id: 'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#crosswalk.csv',
+          label: "Raw data distribution 'crosswalk.csv' file",
+          title: "Raw data distribution 'crosswalk.csv' file",
+          downloadUrl:
+            'https://cdn.humanatlas.io/digital-objects/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2/assets/crosswalk.csv',
+          accessUrl: 'https://lod.humanatlas.io/2d-ftu/kidney-ascending-thin-loop-of-henle/v1.2#crosswalk.csv',
+          mediaType: 'text/csv',
+        },
+      ],
+    },
+  };
+
+  it('should initialize filters from query params', async () => {
+    const { fixture } = await render(MainPageComponent, {
```

Comment:
Create a small `setup` function that does the common logic instead of repeating the same logic in each test